packages/react-components/react-migration-v8-v9/library/package.json

```diff
@@ -37,8 +37,6 @@
     "@fluentui/react-components": "^9.54.13",
     "@fluentui/react-icons": "^2.0.245",
     "@fluentui/react-hooks": "^8.8.12",
-    "@fluentui/react-theme": "^9.1.19",
```

Comment:
non of these packages is used directly in the source code - this was added here historically via template generator.